testing/pom.xml

```diff
@@ -39,6 +39,11 @@
             <artifactId>javax.json-api</artifactId>
             <version>1.1.4</version>
         </dependency>
+        <dependency>
+            <groupId>org.glassfish</groupId>
+            <artifactId>javax.json</artifactId>
+            <version>1.1.4</version>
+        </dependency>
```

Comment:
`<scope>test</scope>`?

Reply:
No, because this isn't for testing `pay-java-commons`, it's for including in the `testing` artefact which is then used in `test` scope by consuming apps

Reply:
That makes sense